beps/0009-plugin-metadata/README.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Plugin Metadata
+status: provisional
+authors:
+  - '@Rugvip'
+owners:
+  - '@backstage/maintainers'
+project-areas:
+  - core
+creation-date: yyyy-mm-dd
+---
+
+<!--
+**Note:** When your BEP is complete, all these pre-existing comments should be removed
+
+When editing BEPs, aim for tightly-scoped, single-topic PRs to keep discussions focused. If you disagree with what is already in a document, open a new PR with suggested changes.
+-->
+
+# BEP: Plugin Metadata
+
+<!-- Before merging the initial BEP PR, create a feature issue and update the below link. You can wait with this step until the BEP is ready to be merged. -->
+
+[**Discussion Issue**](https://github.com/backstage/backstage/issues/NNNNN)
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Design Details](#design-details)
+- [Release Plan](#release-plan)
+- [Dependencies](#dependencies)
+- [Alternatives](#alternatives)
+
+## Summary
+
+<!--
+The summary of the BEP is a few paragraphs long and give a high-level overview of the features to be implemented. It should be possible to read *only* the summary and understand what the BEP is proposing to accomplish and what impact it has for users.
+-->
+
+## Motivation
+
+All Backstage adopters benefit from it being easy to find, install, configure, and validate the setup of plugins and modules. That's regardless of whether one is only using a minimal set of core plugins, or installing a large number of 3rd party features from the Backstage ecosystem. This proposal aims to build upon our [existing package metadata](https://backstage.io/docs/tooling/cli/build-system#package-roles) to introduce a new set of standardized fields that plugins can use to provide additional information about themselves.
+
+This metadata will be used to improve tooling in and around Backstage, such as providing better validation and more powerful utilities in the Backstage CLI, and more information to be surfaced in association with plugins for better discoverability.
+
+### Goals
+
+We want to provide a new set of standardized fields that makes it possible to build tooling that improves the following interactions with the Backstage ecosystem:
+
+- Browsing packages related to a plugin, i.e. listing all available library packages and frontend/backend variants of a specific plugin.
+- Discoverability of plugin modules. Given a plugin you should be able to identify all of its modules.
```

Comment:
Both of these goals ar formulated as if they go "downward", as in, they start with the knowledge of what the plugin is, and then looks outward to things that are effectively part of it. But we're modelling the inverse as well, which makes sense, but not stated as a goal.

Reply:
Yep, added a link back from modules to plugins as well